cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -77,12 +112,9 @@ def test_delete_static_custom_attributes(provider):
         * The attributes was successfully deleted
         (you should not see a custom attributes table)
     """
-    # Checking that all ATTRIBUTES_DATASET was added.
-    attribs_names = [attr.name for attr in provider.custom_attributes()]
-    assert all([attr.name in attribs_names
-                for attr in ATTRIBUTES_DATASET])
+    setup_dataset(provider, verify_exists=True)
 
-    provider.delete_custom_attributes()
+    provider.delete_custom_attributes(*ATTRIBUTES_DATASET)
```

Comment:
setup_dataset() makes the same call, but with the added protection of only passing attributes that are in DATASET and are present. Intentionally skipping that protection here?